src/main/java/step1/model/StringAddCalculator.java

```diff
@@ -10,31 +10,42 @@
     private static final String CUSTOM_DELIMITER_REGEX = "//(.)\\n(.*)";
 
     public static int splitAndSum(String inputString) {
-        if(checkNullinput(inputString)) {
+        if (checkNullinput(inputString)) {
             return 0;
         }
         String[] splitString = makeSplitString(inputString);
         return sumNumbers(splitString);
     }
 
-    private static int checkMinus(String stringNum) {
-        int number = 0;
+    private static boolean checkMinus(int num) {
+
+        if (num < 0) {
+            throw new RuntimeException("양수를 넣어주세");
+        }
+
+        return true;
+    }
+
+    private static int convrrtStringToint(String stringNum) {
+            int number = 0;
+
         try {
```

Comment:
메서드명 오타와 들여쓰기 간격이 맞질 않네요 :)
```suggestion
    private static int convertStringToInt(String stringNum) {
        int number = 0;
        try {
```

Reply:
반영 완료 했습니다.